Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -90,9 +93,8 @@ public void AddRenderer(Renderer _renderer)
                 if (!renderers.Contains(_renderer))
                 {
                     renderers.Add(_renderer);
+                    ToggleClippingFeature(_renderer.EnsureComponent<MaterialInstance>().AcquireMaterials(this), true);
```

Comment:
I noticed that we always turn the keyword on now. Does this still work as expected when a renderer is added to a inactive clipping primitive (ie the renderer should not be clipped until the clipping primitive is activated)?